handlers/namespaces.go

```diff
@@ -46,13 +46,31 @@ func NamespaceHealth(w http.ResponseWriter, r *http.Request) {
 	if rateIntervals, ok := queryParams["rateInterval"]; ok && len(rateIntervals) > 0 {
 		rateInterval = rateIntervals[0]
 	}
+	healthType := "app"
+	if healthTypes, ok := queryParams["type"]; ok && len(healthTypes) > 0 {
```

Comment:
I think it would be good to add documentation on this endpoint and specially about this param.